javascript-timers/interval.js

```diff
@@ -0,0 +1,15 @@
+var $countdown = document.querySelector('.countdown-display');
+var count = 4;
+function countdownDisplay() {
+  $countdown.innerHTML = count--;
+  if ($countdown.textContent === '0') {
+    $countdown.textContent = '~Earth BeeeelooowwwUs~';
+  }
+  if ($countdown.textContent === '~Earth BeeeelooowwwUs~') {
+    clearInterval(timerId);
+  }
```

Comment:
Instead of having an additional conditional, you should just do this in the same conditional block that you set the text content.